src/app/organizations/organization/organization.component.ts

```diff
@@ -66,4 +66,17 @@ export class OrganizationComponent implements OnInit {
       data: { description: description }, width: '600px'
     });
   }
+
+  /**
+   * The database stores a gravatar link if the image was added using UI, but this is not the case if added
+   * via the API.
+   */
+  genGravatarUrl(url: string): Observable<string> {
```

Comment:
This function is executed a few too many times (32 or so times for me) and should be in a service.  Wrapping it in an observable and async pipe isn't actually fixing it.

Reply:
Redid in latest commit. Only gets called once now.